client/my-sites/checkout/checkout-thank-you-i2/masterbar/default-contact.tsx

```diff
@@ -0,0 +1,81 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
Just copying over the component from Marketplace, maybe we should clean it up or maybe not?